### DISCUSSION TITLE
accruedclaims PlacerHolderAPI won't load the Bonus claims i add

### DISCUSSION THREAD

**Discussion by NxTPanic** [GriefPrevention/GriefPrevention#1602] [General]
Dear Folks!

The cacruedclaims PlacerHolderAPI won't load the Bonus claims i add to a player is it possible that the accruedclaims PlacerHolderAPI load sthe Bonus claims i add?

The placeholder remainingclaims load them in.

&d {placeholderapi_griefprevention_remainingclaims_formatted} &d/ placeholderapi_griefprevention_accruedclaims_formatted}

![image](https://user-images.githubusercontent.com/24211237/134974791-6d1fab52-4824-46fe-b56d-a0736e9925c0.png)
![image](https://user-images.githubusercontent.com/24211237/134974808-b3c56200-77fe-42c9-bc81-4738a43ee51d.png)

**Comment by Jikoo** [comment]
If you're looking for some change to the GP PAPI integration, you should make requests [where it is maintained](https://github.com/pixar02/Expansions).

I think you're asking for a new placeholder that combines accrued and bonus claim blocks to form total claim blocks, but your sentence is so garbled that I'm really not sure even after several rereads. Please consider a proofread and maybe a rephrase while opening your issue over there.

**Comment by NxTPanic** [comment]
I'm asking for a new placeholder that combines accrued and bonus claim blocks to form total claim blocks. That is correct. 
There is also a way in FeatherBoard to combine placeholders  correct  me if i'm wrong?

**Reply by Jikoo** [reply]
I have no idea, I've never worked with Featherboard or PAPI. Probably best to seek help for that in Featherboard's support if you think that's a feature. If not, I already linked the GP PAPI repo, you can request the new placeholder there.
